src/extension/providers/webbtc/index.ts

```diff
@@ -7,12 +7,12 @@ declare global {
 }
 
 export default class WebBTCProvider extends ProviderBase {
-  isEnabled: boolean;
+  isWebBTCEnabled: boolean;
```

Comment:
this should be undone, right?

Reply:
this conflicts with the function name isEnabled. so I changed boolean name

Reply:
Edit: both this and executing should be removed

Reply:
resolved